frameit/lib/frameit/options.rb

```diff
@@ -44,6 +44,11 @@ def self.available_options
                            is_string: false,
                            description: "Use iPhone 6s frames instead of iPhone 7 frames",
                            default_value: false),
+        FastlaneCore::ConfigItem.new(key: :use_legacy_iphonex,
```

Comment:
Need to remove since no iPhone XS frames yet 😔

Reply:
I have the frames too, if you are eager to push that. Facebook won't add those soon.

Reply:
@mirkokg How do you have the frames??? 🤔 Are they the same style of frames that we pull from https://facebook.design/device ???

Reply:
Our designer prepared the frames and we are already using those in our release cycle, including deliver and frameit changes that I pushed. I also have frameit offsets. We didn't need all of the colors for XR I can ask him to prepare those as well.

![apple iphone xs max silver](https://user-images.githubusercontent.com/1269927/45805359-dbaa9d80-bcbe-11e8-9957-296d153bf226.png)